plugins/@grouparoo/intercom/src/lib/export-contacts/exportProfile.ts

```diff
@@ -287,14 +292,16 @@ async function deleteUser(
   newGroups = [];
   await updateTags(client, cacheData, user, oldGroups, newGroups);
 
+  if (!syncOperations.delete) {
+    throw new InfoError("Destination is not removing contacts.");
```

Comment:
👍 on keeping this an `InfoError` - we don't want these types of errors to scare users / be retried.